enterprise/templates/enterprise/grant_data_sharing_permissions.html

```diff
@@ -1,113 +1,115 @@
-<%namespace name='static' file='/static_content.html'/>
-  <%static:css group='style-vendor'/>
-    <%! main_css = "style-main-v2" %>
-<%static:css group='${self.attr.main_css}'/>
-  <%!
-    from django.utils.translation import ugettext as _
-    %>
-<%include file="widgets/segment-io.html" />
-<html lang="${ page_language }">
-  <title>${ title_bar_prefix } | ${ platform_name }</title>
+{% load i18n theme_pipeline staticfiles %}
+
+{% stylesheet 'style-vendor' %}
+{% stylesheet 'style-main-v2' %}
+
+{% include "widgets/segment-io-django.html" %}
+
+<html lang="{{ page_language }}">
+  <title>{{ title_bar_prefix }} | {{ platform_name }} </title>
   <head>
-    <link rel="stylesheet" href="${static.url('css/vendor/font-awesome.css')}"/>
-    <link rel="stylesheet" href="${static.url('enterprise/grant_data_sharing_permissions.css')}"/>
+    <link rel="stylesheet" href="{% static 'css/vendor/font-awesome.css' %}"/>
```

Comment:
Can we break this template into a shared `base.html` which contains the HTML skeleton including any common static files that will be needed on all pages and a separate template that implements the course enrollment landing page and extends the base template?